spec/features/admin_views_admin_panel_spec.rb

```diff
@@ -1,37 +1,41 @@
 require 'rails_helper'
 
-feature 'user views homepage', %Q{
+feature 'admin views admin panel', %Q{
   As an admin, I want to view the admin panel, so that I can see what tools are available to me.
   Acceptance Criteria:
-  * [ ] - I can view the main page of the admin panel only if I am logged in
+  * [X] - I can view the main page of the admin panel only if I am logged in
           and have admin access.
-  * [ ] - If I am not logged in or don’t have admin access, I can’t see a link
+  * [X] - If I am not logged in or don’t have admin access, I can’t see a link
           to the admin panel in the side bar.
-  * [ ] - If I am not logged in or don’t have admin access, I get a 404 error
+  * [X] - If I am not logged in or don’t have admin access, I get a 404 error
           when visiting the admin namespace.
   } do
 
-  let(:user) do
+  let(:user1) do
     FactoryGirl.create(:user)
   end
 
-  scenario 'unauthenticated user', focus: true do
-    visit '/admin'
+  let(:admin) do
+    FactoryGirl.create(:user, admin: true)
+  end
+
+  before { Capybara.reset_session! }
 
+  scenario 'unauthenticated user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.